README.md

```diff
@@ -74,21 +74,86 @@ Here's the [documentation site](https://stackotter.github.io/swift-cross-ui/docu
 
 ### macOS: Installing Gtk 4.10+
 
-Install Gtk 4.10+ using homebrew or the package manager of your choice.
+Install Gtk 4.10+ using HomeBrew or the package manager of your choice.
 
 ```sh
 brew install gtk4
 ```
 
-### Linux: Installing Gtk 4.10+ and clang
+### Linux: Installing Gtk 4.10+ and Clang
 
-Install Gtk+3 and Clang using apt or the package manager of your choice. Ensure that the installed
+Install Gtk+ and Clang using apt or the package manager of your choice. Ensure that the installed
 version of Gtk is Gtk 4.10+
```

Comment:
Just noticed this one is also wrong. We should maybe also add that gtk is probably already installed for Linux users with GNOME environment? @stackotter

Reply:
Yeah sure 👍